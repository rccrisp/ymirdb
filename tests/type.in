set a 1
set b 2
type b
push b a
type b
pop b
type b
append b a
type b
pluck b 2
type b
set b a
type b
pop b
type b
bye
