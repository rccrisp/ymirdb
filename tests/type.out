> ok

> ok

> simple

> ok

> general

> a

> simple

> ok

> general

> a

> simple

> ok

> general

> a

> simple

> bye
